pkg/addons/data/ClusterRoleBinding.rbac.authorization.k8s.io/dedicated-admin-reconciler.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
It can probably be a RoleBinding instead of a ClusterRoleBinding?

Reply:
Removing the clusterrolebinding permission.